classes/Task/Upgrade/UpgradeNow.php

```diff
@@ -53,45 +53,26 @@ public function run(): int
 
         $this->container->getWorkspace()->createFolders();
 
-        $channel = $this->container->getUpgradeConfiguration()->get('channel');
         $upgrader = $this->container->getUpgrader();
 
-        $upgrader->branch = VersionUtils::splitPrestaShopVersion(_PS_VERSION_)['major'];
-        $upgrader->channel = $channel;
-        if ($this->container->getUpgradeConfiguration()->get('channel') == 'private' && !$this->container->getUpgradeConfiguration()->get('private_allow_major')) {
-            $upgrader->checkPSVersion(false, ['private', 'minor']);
-        } else {
-            $upgrader->checkPSVersion();
-        }
-
         if ($upgrader->isLastVersion()) {
             $this->next = '';
-            $this->logger->info($this->translator->trans('You already have the %s version.', [$upgrader->version_name]));
+            $this->logger->info($this->translator->trans('You are using the most up to date version for your PHP version'));
```

Comment:
Can we find something more generic? We may look for updates based on something else than the PHP version in the future.

Reply:
"Your shop is currently running the latest compatible version. No updates are needed at this time."